tests/backends/pycryptodome/test_results.py

```diff
@@ -0,0 +1 @@
+../test_results.py
```

Comment:
?? what am I looking at?

Reply:
symlink :p

I was playing with this approach in order to keep shared tests under the /tests folder, rather than in eth_hash/utils/, which feels a little odd to me.

Do you think any development environments we care about won't support it? Ofc open to other ideas.

Reply:
> what I mean by counter-intuitive is that someone might edit this file, not realizing they were editing the file for all of the backends.

That's helpful, I'll throw in a comment at least.